README.md

```diff
@@ -46,8 +46,10 @@ Example:
 ### Daemonset
 Checks if a specified daemonset is already running on the same host, this dependency requires a `POD_NAME`
 env which can be easily passed through the [downward api](http://kubernetes.io/docs/user-guide/downward-api/).
+The `POD_NAME` variable is used to determine right host to check dependencies.
```

Comment:
I would change it to :
The `POD_NAME` variable is **mandatory** and is used to resolve dependencies.